### DISCUSSION TITLE
Clock Syncing Issues

### DISCUSSION THREAD

**Discussion by izaghast** [djphazer/O_C-Phazerville#107] [Q&A]
Hello!

 I am trying to get the internal clock synced up with the external clock from my trigger sequencer. I'm having a problem where the internal clock starting slightly later than my sequencer. For example, on DrumMap, the triggers are one 16th note late. Step one on DrumMap occurs when it is step 2 on my sequencer. I can work around this by offsetting my trigger sequencer patterns by one step, but I was just curious if there was any way to change where the first beat lands in the O_C?

Thank you!

**Comment by djphazer** [comment]
What version are you running? Seems like this was a bug I've encountered and possibly fixed...

**Reply by djphazer** [reply]
Otherwise, tell me more details about the internal clock configuration (screenshot maybe) - and are you using TR1 for external Clock? external reset or no?

**Reply by izaghast** [reply]
Thank you for the quick reply! I am on version 1.8, not the most recent, so that may be an issue. I have attached images of the applets and settings below! The only thing sent into the input is a clock for every 16th note.

 ![image](https://github.com/user-attachments/assets/f11778c9-2364-4cf4-90a0-30050f181869)![image](https://github.com/user-attachments/assets/dfcef0f6-04ee-409a-9bbd-b3b1082ecc41)

**Comment by djphazer** [comment]
Aha! I reproduced it using an external Reset trigger. If you do this, the problem is sometimes the external trigger arrives _after_ the internal clocks already fired.

~~You must use internal triggers or external triggers but not both on the same applet!~~ DrumMap already has auto-reset anyway ;)
Edit: it can also happen with two external triggers, if the Reset arrives too late. One solution would be to delay the external Clock signal just a lil bit.

**Reply by izaghast** [reply]
Apologies for the late reply, I had updated my module, and the problem still persisted. I have figured out that my sequencer does not send a clock pulse out on step 1, causing the O_C's clock to start on step 2 of my sequencer. Do you know of any work around for this? any way to start DrumMap on a different step? If not, that is understandable, and I may need a different external clock source.

Thank you!

**Reply by djphazer** [reply]
What is the external sequencer? Maybe it can be fixed... 🤔

**Reply by izaghast** [reply]
Its from a Step Fader MkII from Tesseract Modular. I will reach out to them to see what they say!
